src/status_im/ui2/screens/chat/composer/mentions.cljs

```diff
@@ -0,0 +1 @@
+(ns status-im.ui2.screens.chat.composer.mentions)
```

Comment:
cc @OmarBasem

Reply:
I hope this will not cause a circular dependency. 
The mentions PR is merged.